ciao-cli/image.go

```diff
@@ -251,8 +264,14 @@ func (cmd *imageListCommand) run(args []string) error {
 		}
 	}
 
-	url := buildImageURL("images")
-	resp, err := sendHTTPRequest("GET", url, nil, nil)
+	var url string
```

Comment:
This could could be factorized out into a separate function.  It's used in 5 different places.